sources/packages/backend/apps/api/src/services/notification/gc-notify.service.ts

```diff
@@ -22,9 +22,7 @@ export class GCNotifyService {
    * @param payload
    * @returns GC Notify API call response.
    */
-  async sendEmailNotification<T>(
-    payload: RequestPayload<T>,
-  ): Promise<GCNotifyResult> {
+  async sendEmailNotification<T>(payload: unknown): Promise<GCNotifyResult> {
```

Comment:
I believe that there is no need for the "<T>" here, right?

Reply:
Removed.